CONTRIBUTING.md

```diff
@@ -14,6 +19,9 @@ https://docs.docker.com/engine/install
 
 How to install Make
 sudo apt-get install build-essential
```

Comment:
@extrawurst is `Make` needed at all?

Reply:
yeah unfortunately we have one last Makefile in `backend-e2e` - worth opening another issue to clean that up though and migrate it to just too

Reply:
👍 addressed by opening #84